examples/tutorials/Introduction_to_PROTACs.ipynb

```diff
@@ -0,0 +1,1908 @@
+        "\\\\\n",
+        "\n",
+        "#### 1.1.2 Connection to PROTACs\n",
+        "\n",
+        "The realization that the UPS could be leveraged for therapeutic purposes was initially made through early studies of viruses and plants. The underlying idea involves design of small molecules capable of recruiting the E3 ligase and inducing degradation of a protein of interest (POI). This general idea naturally extended itself to the case of PROTACs. Together, the POI ligand, linker, and E3 ligase ligand make up the PROTAC complex responsible for protein degradation. Note that the presence of two ligands enables simultaneous  recruitment of the E3 ligase and POI, hence its heterobifunctionality property.\n",
+        "\n",
+        "Furthermore, after the POI is degraded by the proteasome, PROTACs can disassociate and continue to induce further degradation, enabling low concentrations to be efficacious. This catalytic mechanism of action and event-drive pharmacology prevents PROTACs from suffering the same limitations as conventional therapeutic strategies such as drug resistance and off-target effects.\n",
+        "\n",
+        "![protac.png](https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41392-019-0101-6/MediaObjects/41392_2019_101_Fig1_HTML.png?as=webp)"
+      ],
+      "metadata": {
+        "id": "IlR0JZyU4PVe"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### 1.2 How can we leverage machine learning?\n",
+        "\n",
+        "As a novel and promising technique, PROTACs have demonstrated positive clinical results thus far. However, much of the clinical validation has been against classically drugged targets. In order for PROTACs to reach their full potential, their clinical efficacy against novel or hard to reach targets must be demonstrated. Consequently, there has been growing research in designing PROTAC molecules capable of elicting an effective response. However, much of the current work is empirical and requires extensive trial-and-error processes. Machine learning could potentially revolutionize this. By correlating molecular structure with physiochemical properties and biological activity, we could potentially streamline the discovery process, significantly reducing the time and cost associated with validation. With that in mind, let's jump into this tutorial to predict efficacy of PROTAC degraders!\n",
+        "\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "LYSEBoaq4VBr"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "For a more in-depth dive into PROTACs, ubiquitin proteasome system, and targeted protein degradation, readers are referred to \"[PROTAC targeted protein degraders: the past is prologue](https://www.nature.com/articles/s41573-021-00371-6)\" and \"[An overview of PROTACs: a promising drug discovery paradigm](https://link.springer.com/article/10.1186/s43556-022-00112-0).\"\n"
```

Comment:
I will add a phrase saying the main purpose of the tutorial and a small outline. 
Data cleaning. 
Featurization. 
Prediction